src/observer/reports.py

```diff
@@ -97,7 +114,7 @@ def format_report(report, format, context):
         RSS: rss.format_report,
         CSV: csv.format_report,
     }
-    ensure(format in report[SERIALISATIONS], "unsupported format %r for report %s" % (format, report['title']))
+    ensure(format in report['serialisations'], "unsupported format %r for report %s" % (format, report['title']))
```

Comment:
Would normally avoid using builtins (`format`) as var names even if they are in an isolated function that doesn't use said builtin. Clear enough you are not using the builtin function here but later on, in the tests, I had to do a second take.

Reply:
fair enough. I don't use format at all so I never picked it up. And neither did the linter for that matter ...

Reply:
turns out this sort of shadowing is not supported by linters, which is surprising.

I've fixed this instance and will keep an eye out in future for you